packages/dataprovider/src/buildVariables.ts

```diff
@@ -182,20 +194,23 @@ const buildNewInputValue = (
         (t) => t.name === fieldObjectType.name,
       ) as IntrospectionInputObjectType;
 
-      // if it has a set modifier, it is an update array
-      const createModifier = fullFieldObjectType?.inputFields.find(
-        (i) => i.name === "create",
-      );
-      const connectModifier = fullFieldObjectType?.inputFields.find(
-        (i) => i.name === "connect",
-      );
+      const isRelationship = fullFieldObjectType?.inputFields.every((i) => {
+        return Object.keys(ModifiersParams).includes(i.name);
+      });
```

Comment:
so this is true, when the field has every of these modifiers?

Reply:
also if the input would have additional fields, this would be false.

Reply:
@macrozone yes, this will be true only if the input contains any (but not all) of prisma modifiers. If it has other fields than the input is not a relationship